packages/react/src/switch/root/SwitchRoot.test.tsx

```diff
@@ -262,7 +262,139 @@ describe('<Switch.Root />', () => {
 
   describe('Form', () => {
     // FormData is not available in JSDOM
-    it.skipIf(isJSDOM)('should include the switch value in form submission', async () => {
+    it.skipIf(isJSDOM)(
+      'should include the switch value in form submission, matching native checkbox behavior',
```

Comment:
This doesn't actually check if the `value` is submitted

Reply:
`Switch` only supports `"on"`